sdk/core/core-util/review/core-util.api.md

```diff
@@ -17,6 +17,14 @@ export interface AbortOptions {
     abortSignal?: AbortSignalLike;
 }
 
+// @public
+export function calculateExponentialDelayInterval(retryAttempt: number, config: {
```

Comment:
Not loving the name, definitely open to ideas

Reply:
Once I get all feedback addressed I'll port this over to ts-http-runtime (if needed) - just waiting until API is finalized